Source/ACE.Server/Physics/Common/GfxObj.cs

```diff
@@ -33,21 +33,21 @@ public GfxObj(DatLoader.FileTypes.GfxObj gfxObj)
             ID = gfxObj.Id;
             VertexArray = gfxObj.VertexArray;
 
-            Polygons = new Dictionary<ushort, Polygon>();
-            foreach (var kvp in gfxObj.Polygons)
-                Polygons.Add(kvp.Key, PolygonCache.Get(kvp.Value, gfxObj.VertexArray));
+            //Polygons = new Dictionary<ushort, Polygon>();
```

Comment:
why are these commented out?

this will break a ton of stuff...

Reply:
recommended to place these commented out blocks surrounded by this if check:

            if (!PhysicsEngine.Instance.Server)
            {
                Polygons = new Dictionary<ushort, Polygon>();
                foreach (var kvp in gfxObj.Polygons)
                    Polygons.Add(kvp.Key, PolygonCache.Get(kvp.Value, gfxObj.VertexArray));
            }

Reply:
done.